script/ablist.yml

```diff
@@ -436,14 +446,19 @@
     - slim
   inconsiderate:
     - anorexic
+    - bony
+    - skinny
```

Comment:
Hmm, what was the reason to add skinny here?

Reply:
well , good question , the thing is , skinny comes from a description that someone is "skin on bones" which is used to reference people in late stages of starvation , though of course recently it has been normalized in day to day English as a casual word , however, if we are to level this description at someone who actually has anorexia for example , I thought it sounded inconsiderate in that context , or you know , I could just be overthinking this  :'D.
But yeah , here  is how my thought process went for both "bony" and "skinny" .
Anyways , I'll be removing skinny now .

Reply:
Also concerning the upper case in `race.yml` , I just capitalized the races ,regions , nationalities and whatnot , I wasn't aware the tooling puts everything to lower case before  matching , so I 'll be on it :+1:

Reply:
Finally , it's my pleasure , mate ! :+1: